client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -163,6 +168,28 @@ const GSuiteNewUser: FunctionComponent< Props > = ( {
 
 						{ hasMailBoxError && <FormInputValidation text={ mailBoxError } isError /> }
 					</div>
+
+					{ config.isEnabled( 'gsuite/passwords' ) && (
+						<div className="gsuite-new-user-list__new-user-password-container">
```

Comment:
I noticed that they browser keeps popping up with a modal to save the passwords. Since this is only an entry UI for account/user creation and most admins may create more than one user, should we attempt to prevent that pop-up?

Reply:
I see your point but maybe some users would actually like to save those passwords somewhere. Also, how would you prevent that?